src/sass/modules/_m-vet-nav.scss

```diff
@@ -459,6 +459,7 @@ body.va-pos-fixed {
 
 @include media($medium-screen) {
   .child-menu-opened {
+    height: auto;
```

Comment:
This allows us to use initial if it is supported, and auto otherwise. 
An issue created to address all such issues has been created [here](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/8672).

Reply:
What do you think about getting rid of the `height: initial` below? If I remember right, the `height: auto` should do the same thing in this case, yeah?

Reply:
Sounds good, changed.